packages/one-app-bundler/bin/bundle-module.js

```diff
@@ -18,16 +18,20 @@ const path = require('path');
 const fs = require('fs');
 const localeBundler = require('@americanexpress/one-app-locale-bundler');
 
+const getConfigOptions = require('../utils/getConfigOptions');
 const clientConfig = require('../webpack/module/webpack.client');
 const serverConfig = require('../webpack/module/webpack.server');
 const getWebpackCallback = require('./webpackCallback');
 const { watch } = require('../utils/getCliOptions')();
 
 const modernClientConfig = clientConfig('modern');
 const legacyClientConfig = clientConfig('legacy');
+const configOptions = getConfigOptions();
+const disableDevelopmentLegacyBundle = configOptions.disableDevelopmentLegacyBundle && process.env.NODE_ENV === 'development';
```

Comment:
```suggestion
const disableDevelopmentLegacyBundle = getConfigOptions().disableDevelopmentLegacyBundle && process.env.NODE_ENV === 'development';
```
Since its not being used below do we need to create a var ?